lein-light-nrepl/src/lighttable/nrepl/eval.clj

```diff
@@ -70,7 +81,9 @@
      (and (string? res) (:verbatim opts)) res
      :else (pr-str res))))
 
-(defn truncate [v]
+(defn truncate
+  "Useless function... Same as 'identity'. Deprecated and to be removed."
```

Comment:
Should we keep "Same as 'identity'. Deprecated and to be removed." at least? Letting users know the function is to be deprecated and that there already exists an alternative seems reasonable.